nntrainer/layers/layer_node.cpp

```diff
@@ -188,19 +190,20 @@ const std::string LayerNode::getName() const noexcept {
 std::ostream &operator<<(std::ostream &out, const LayerNode &l) {
 
   auto &input_layers =
-    std::get<std::vector<props::InputLayer>>(*l.layer_node_props);
+    std::get<std::vector<props::InputConnection>>(*l.layer_node_props);
 
   out << "[" << l.getName() << '/' << l.getType() << "]\n";
   auto print_vector = [&out](const auto &layers, const std::string &title) {
```

Comment:
minor comment: might be clearer to define type of object `layers` here or give the variable a clearer name.

Reply:
Yeah, I think this has been handled at the upcoming PR, if not I'll add a change :)

Reply:
cool.